src/test/java/ui/OmeZarrS3V4Opener.java

```diff
@@ -10,9 +14,10 @@
 public class OmeZarrS3V4Opener {
```

Comment:
As far as I can see there are no tests yet in this class?
Like `assertEquals(...)`...
Is this on purpose or did you want to add some?

Reply:
It is a UI test (only for visual check), assertEquals not needed